imagebuildah/executor.go

```diff
@@ -245,6 +247,10 @@ func NewExecutor(store storage.Store, options BuildOptions, mainNode *parser.Nod
 			fmt.Fprintf(exec.out, prefix+format+suffix, args...)
 		}
 	}
+
+	// If we have a value from the '--from' option, remember it.
+	exec.fromOverride = options.From
```

Comment:
Why set it here instead of on line 233?

Reply:
Because at the time my noggin was thinking I needed to set it to "" by default, but that I already took care of in the declaration in pkg/cli/common.go.  Good thought, I'll change it.